wait_any.go

```diff
@@ -0,0 +1,71 @@
+package asynctask
+
+import (
+	"context"
+	"fmt"
+)
+
+// WaitAnyOptions defines options for WaitAny function
+type WaitAnyOptions struct {
+	// FailFast set to true will indicate WaitAny to return on first error it sees.
+	FailFast bool
```

Comment:
```suggestion
        // Set to true if you want to fail the WaitAny, otherwise we only fail WaitAny when all task fails.
	FailOnAnyError bool
```